lib/tasks/gather.js

```diff
@@ -177,8 +177,10 @@ class TaskGather extends SttTask {
     this._startVad();
 
     const startDtmfListener = () => {
+      if (this._dtmfListenerStarted) return;
```

Comment:
Is there a scenario where this gets called twice?

Reply:
You are right, we don't have a case that this startDtmfListener is called twice.

Reply:
In that case can you make it an assert instead so if it ever does happen we get an error we can fix
```
assert(! this._dtmfListenerStarted);
```